### ISSUE TITLE
[Bug] Blue screen of death after not allowing eac.exe (Easy AntiCheat) then all filters removed.

### ISSUE THREAD

**Issue by danielbc666666** [henrypp/simplewall#1741]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

3.7.8

### Windows version

Windows 11 22H2

### Steps to reproduce

![simplewall_ThvVQFhET4](https://github.com/henrypp/simplewall/assets/80385117/8ba50910-bc30-4f21-8469-124f484e642c)
![Signal_t28LWleHJU](https://github.com/henrypp/simplewall/assets/80385117/43bdcfe4-814a-496f-add4-cefb6160c9db)

### Expected behavior

Basically I installed the game For Honor that needs to run Easy Anticheat. After running the game Simplefirewall asked if i wanted to allow eac.exe. I didn't answer 

### Actual behavior

I instantly didn't answer to the game left keep opening, after 5 seconds windows crashed.
This didn't happen anymore after allowing eac.exe.
The weird part is that i didn't allow eac.exe in other games like Brawlhalla or Elden Ring and had 0 problems.

**Comment by henrypp** [comment]
and it sw problem that another app bsoded?

**Comment by danielbc666666** [comment]
My concern in all this is that after restarting the computer all the filters that I had in simplefirewall disappeared

**Comment by henrypp** [comment]
so, isstall it yorselfs
